modules/integration_azure_firewall/common-modules.tf

```diff
@@ -0,0 +1 @@
+../../common/module/modules.tf
```

Comment:
you should remove this link which works for smart_agent module only and cp one from one existing azure module (I should document this better)